### ISSUE TITLE
[Bug]: Need to improve the design and functionality of nav bar and side bar

### ISSUE THREAD

**Issue by Latish705** [AppointNext/AppointNext#5]
![image](https://github.com/AppointNext/AppointNext/assets/104522651/909485b1-6a79-495e-969c-b2d68b2510ce)

Currently the nav is overlap the other components

Some new design that could increase ui/ux of users

**Comment by mBasak03** [comment]
Can I give it a try?

**Comment by Latish705** [comment]
@mBasak03 Yeah sure are participating under and program?

**Comment by mBasak03** [comment]
> @mBasak03 Yeah sure are participating under and program?

no, I am a beginner, learning about open source contribution, planning to participate in GSOC next year

**Comment by Latish705** [comment]
Okay ask if you need any help

**Comment by mBasak03** [comment]
> Okay ask if you need any help

Can you please tell me from where I can see the navbar? and Exactly where I can see the error?
![image](https://github.com/AppointNext/AppointNext/assets/163230793/504acab3-917b-40a3-81b6-0255528c59d3)

**Comment by Latish705** [comment]
First you need to sign in for test purposes 

Usename - testn12
Pass - 1234

Then you can see the navbar

**Comment by mBasak03** [comment]
I think there is some error with the login functionality:
![image](https://github.com/AppointNext/AppointNext/assets/163230793/b52d7222-f54d-44c4-be5b-e9437987810b)


and  I tried to set the backend locally, but it is showing me an error like this :
 
![image](https://github.com/AppointNext/AppointNext/assets/163230793/2c52eaba-df69-4017-8b3d-35121d4ebec1)

I never used Django before. So Can you help me to set this backend?

**Comment by Latish705** [comment]
@mBasak03 it says you not installed the django

**Comment by Latish705** [comment]
Have you done pip install and you can also skip migrate and run directly

**Comment by vishnupriyagithub** [comment]
Hey I would like to work on this issue as a part of  GSSOC'24 Extended. Please assign me

**Comment by Latish705** [comment]
@vishnupriyagithub go ahead assigning it you

**Comment by vishnupriyagithub** [comment]
Could you add Gssoc extended label and level

**Comment by vishnupriyagithub** [comment]
hey the backend and frontend are running fine but I'm facing issue with login feature(prolly connectivity with backend). Could you please give some info

**Comment by Latish705** [comment]
@vishnupriyagithub share ss

**Comment by vishnupriyagithub** [comment]
@Latish705 
![Screenshot 2024-10-02 122702](https://github.com/user-attachments/assets/b05210bf-0b56-4f37-9778-a0ffb307cded)
there is some issue authentication i guess
server logs:
![Screenshot 2024-10-02 123132](https://github.com/user-attachments/assets/49080199-615f-4c5b-af64-fab5df52af9c)

![Screenshot 2024-10-02 115115](https://github.com/user-attachments/assets/28c4995f-949f-47de-b2a5-370fbf87aa6e)
